validation/models.py

```diff
@@ -355,6 +364,26 @@ def generate_primary_key(sender, instance, **kwargs):
 SHA256_HEX_LENGTH = 64
 
 
+class PresentableRecordingModelManager(models.Manager):
+    """
+    Returns Recording instances only for recordings we think are
+    clean/usable to be presented to users of e.g., the dictionary.
+    """
+
+    def get_queryset(self) -> "QuerySet[Recording]":
+        recordings = super().get_queryset()
+        return (
+            recordings.exclude(quality=Recording.BAD)
+            .exclude(wrong_word=True)
+            .exclude(wrong_speaker=True)
+            # We use the "gender" field as a proxy to see whether a speaker's data has
+            # been properly filled out: exclude speakers whose gender field has not been
+            # input. An admin must put *something* in the gender field before the
+            # speaker shows up in API results.
+            .exclude(speaker__gender__isnull=True)
+        )
+
+
```

Comment:
I'm okay with having this one in `views.py` as that's where I do a lot of other query filtering and I find it easier to have it all in one file. I do understand how this could be cleaner though.